src/modules/common/components/paginator/paginator.styles.less

```diff
@@ -33,6 +35,30 @@
   height: 100%;
   padding: 0;
   text-decoration: none;
+
+  i {
+    align-items: center;
+    background: @color-theme-primary;
+    border: 1px solid @color-highlight;
+    border-radius: 3px;
+    box-sizing: border-box;
+    display: flex;
+    height: 32px;
+    justify-content: center;
+    width: 32px;
```

Comment:
are `px` required here or can `rem` be used?

Reply:
nice catch